index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile_website</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="wrapper">
+    <div class="wrapper-1">
+      <header>
+        <div class="logo"><a href="">My Logo</a></div>
+        <nav>
+          <a href="">
+            <img id="menu" src="./images/menu.png" alt="menu">
+          </a>
+
+        </nav>
+
+      </header>
+      
+    </div>
+    <section class="content">
+      <h1>I’m Kaweesi  Glad to see you!</h1>
```

Comment:
- [ ] Please increase the font weight for the main heading and make sure it matches the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185659111-f3f8034b-6dff-4579-b878-13ce79617e91.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/185659159-60f57381-f34e-47ae-b387-36ce3c386c10.png)